packages/account/src/Sections/Verification/ProofOfIdentity/onfido.jsx

```diff
@@ -14,13 +14,8 @@ import { onfido_status_codes } from './proof-of-identity';
 const onfido_container_id = 'onfido';
 
 const OnfidoContainer = ({ height }) => {
-    const is_auto_height = !height;
-    const height_props = {
-        ...(is_auto_height && { autoHeight: true }),
-        ...(!is_auto_height && { height }),
-    };
     return (
-        <ThemedScrollbars is_native={isMobile()} autoHide {...height_props}>
+        <ThemedScrollbars is_native={isMobile()} height={height}>
```

Comment:
:+1: This is much simpler. 
Is it necessary still to send `is_native` tho? Aren't all scrollbars native from now on?

Reply:
Yep, but for `mobile` there's no need to have any Themedscrollbars component used on touch devices. Maybe I'll rename the `is_native` prop to `is_disabled` instead if it makes more sense.

Reply:
@zaki-binary I'd suggest `is_inactive` rather than `is_disabled`, or maybe that's inaccurate too, so rather something that expresses the scrollbar is bypassed rather than the functionality is blocked.

Reply:
I'm open to any suggestions.

Reply:
I'm going for `is_bypassed` for now, unless there are any objections.